SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -17,7 +18,13 @@ namespace SonarRunner.Shim
 {
     public class SonarRunnerWrapper : ISonarRunner
     {
-        private const int SonarRunnerTimeoutInMs = System.Threading.Timeout.Infinite;
+        private const string SonarOptsVariable = "SONAR_RUNNER_OPTS";
```

Comment:
"SonarRunnerOptsVariableName" - "Variable" is ambiguous about whether it is the value of the variable or the name of the variable.